### DISCUSSION TITLE
AppImage location?

### DISCUSSION THREAD

**Discussion by JeanCarloEM** [filecoin-project/lotus#7857] [Build and Sync] [answered]
Sorry, but the official product page, https://lotus.filecoin.io/docs/set-up/install/, says Lotus AppImage is available in releases of github, however, I couldn't locate the build. Can you guide me?

**Comment by SgtCoin** [comment, marked as answer]
@JeanCarloEM We are working on app images for the latest RC releases, however the latest we currently have is for version 1.13.0.

https://github.com/filecoin-project/lotus/releases/tag/v1.13.0

A PR request has been opened to change the documentation. 

https://github.com/filecoin-project/lotus-docs/pull/43
